cli/src/base/ContentDirectoryCommandBase.ts

```diff
@@ -101,11 +83,11 @@ export default abstract class ContentDirectoryCommandBase extends RolesCommandBa
     return selectedClass
   }
 
-  async classEntryByNameOrId(classNameOrId: string): Promise<[ClassId, Class]> {
-    const classes = await this.getApi().availableClasses()
-    const foundClass = classes.find(([id, c]) => id.toString() === classNameOrId || c.name.toString() === classNameOrId)
+  async channelEntryById(channelId: string): Promise<[ChannelId, Channel]> {
```

Comment:
I think now you can just use `channelById` api query, this was just not possible with classes.

Reply:
Yeah, removed it in further PR, done here https://github.com/Joystream/joystream/pull/2258/commits/91fa8e99eb0d25e4f1f2b2bbb6b504860634bfc3